ui/src/app/cohort-search/search-group-item/search-group-item.component.tsx

```diff
@@ -216,7 +216,7 @@ export const SearchGroupItem = withCurrentWorkspace()(
     remove() {
       triggerEvent('Delete', 'Click', 'Snowman - Delete Criteria - Cohort Builder');
       this.updateSearchRequest('status', 'pending', true);
-      const timeout = setTimeout(() => {
+      const timeout = global.setTimeout(() => {
```

Comment:
Here, `setTimeout` can't be `window.setTimeout` because typescript checking gives error.
<img width="802" alt="Screen Shot 2021-03-22 at 1 26 42 PM" src="https://user-images.githubusercontent.com/35533885/112032133-70054d80-8b12-11eb-9184-768358e3f0dd.png">